src/main/cpp/src/query_operations/broad_combined_gvcf.cc

```diff
@@ -1006,6 +1013,48 @@ void BroadCombinedGVCFOperator::handle_deletions(Variant& variant, const Variant
           }
         }
       }
+      //GT field
+      if(original_GT_field_ptr)
```

Comment:
Lines 1016-1057: I think this code can be split in several representative functions.

Reply:
Split into 2 functions now